dashboard/app/views/layouts/_race_interstitial.html.haml

```diff
@@ -1,9 +1,9 @@
 #race-modal.modal{'data-backdrop' => 'static', 'data-keyboard' => 'false', style: 'display: none;'}
   .modal-dialog
     .modal-content.no-modal-icon
-      %h2.custom-h2= t('race_interstitial.title')
+      %h2.custom-h2{style: "margin-top: 20px; margin-bottom: 10px; font-size: 1.7em;"}= t('race_interstitial.title')
```

Comment:
Do we have a policy on what should be inline styling versus in inline CSS or external SCSS?

Reply:
Good point, thank you. I've moved this styles to interstitial.scss